mkosi

```diff
@@ -3019,7 +3015,7 @@ def make_minimal_ext4(args: CommandLineArguments, root: str, for_cache: bool) ->
         f: BinaryIO = cast(BinaryIO, tempfile.NamedTemporaryFile(prefix=".mkosi-mkfs-ext4",
                                                                  dir=os.path.dirname(args.output)))
         f.truncate(args.root_size)
-        run(["mkfs.ext4", "-I", "256", "-L", "root", "-M", "/", "-d", root, f.name], check=True)
+        run(["mkfs.ext4", "-I", "256", "-L", "root", "-M", "/", "-d", root, f.name])
 
     with complete_step('Minimizing ext4 root file system'):
         run(["resize2fs", "-M", f.name])
```

Comment:
This also seems like something we should check

Reply:
If those fail, we'd need to die anyway, something the code would have already done, so this is fine. In the future this be coupled with automatic cleanup for those cases?